### PR TITLE
Roles and privileges - Counselling is dependent on ART

### PR DESCRIPTION
**Pull request by solemabrothers**
https://app.asana.com/0/1133167201254913/1166217269516324

### CODE REVIEW

**Comment by dbaluku** [comment]
@solemabrothers  what qualifies the client to move to the 'attended to' list in the counselling page

**Comment by solemabrothers** [comment]
> 
> 
> @solemabrothers what qualifies the client to move to the 'attended to' list in the counselling page

The Provider completes a patient session

File: omod/src/main/resources/apps/poc_homepage_app.json [hunk 1]
```diff
@@ -44,7 +44,25 @@
...
   ]
-},{
+},
+  {
+    "id": "ugandaemr.counselling.clinic.homepageLink",
+    "description": "Linkages to the counselling section",
+    "order": 1,
+    "extensions": [
+      {
+        "id": "ugandaemr.mch.homepageLink",
```
Comment (slubwama) [inline omod/src/main/resources/apps/poc_homepage_app.json]: @solemabrothers Why ugandaemr.mch.homepageLink

**Review by slubwama — CHANGES_REQUESTED** [review]

File: omod/src/main/resources/apps/poc_homepage_app.json (hunk 1 above)
Comment (solemabrothers) [reply omod/src/main/resources/apps/poc_homepage_app.json]: This was over sight, the change has been made

**Review by slubwama — APPROVED** [review]

**Pull request merged**